### PR TITLE
[pigeon] fixed windows across multiple drives

### PR DESCRIPTION
**Pull request by gaaclarke**
fixes https://github.com/flutter/flutter/issues/68583

### CODE REVIEW

File: packages/pigeon/bin/pigeon.dart [hunk 1]
```diff
@@ -9,20 +9,29 @@ import 'dart:isolate';
 import 'package:path/path.dart' as path;
 import 'package:pigeon/pigeon_lib.dart';
 
+/// This creates a relative path from `input` to `from`, the output being a
```
Comment (xster) [inline packages/pigeon/bin/pigeon.dart:12]: from `from` to `input` no?

**Review by xster — COMMENTED** [review]
LGTM

File: packages/pigeon/bin/pigeon.dart (hunk 1 above)
Comment (gaaclarke) [reply packages/pigeon/bin/pigeon.dart]: good call

**Comment by creativecreatorormaybenot** [comment]
@gaaclarke I will give this a try when it is published 👌

**Comment by creativecreatorormaybenot** [comment]
@gaaclarke It works! 🤘🚀 Thanks a lot for your efforts.

**Comment by yjbanov** [comment]
This broke the build.

**Comment by yjbanov** [comment]
https://cirrus-ci.com/build/4612051702120448

**Comment by creativecreatorormaybenot** [comment]
@yjbanov It is `imitation_game` that failed, though.

**Comment by yjbanov** [comment]
Looks like it, but I don't know much about what this PR changes or whether it's related to `imitation_game`. Only sounding the alarm.

imitation_game_failure

**Comment by gaaclarke** [comment]
@yjbanov Yea, looks like the change in analyzer rules started breaking imitation_game, unrelated to this PR.  Looks like ignoring the warning about using `as` is now also a warning haha.  I'll land something to fix it in a bit.

**Pull request merged**